Tailviewer/BusinessLogic/FileExplorer/IFileExplorer.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Tailviewer.BusinessLogic.FileExplorer
+{
+	public interface IFileExplorer
+	{
+		void SelectFile(string path, IProgress<Percentage> progressReporter = null);

```

Comment:
What's the progress for? It could only toggle from 0 to finished, no?

Reply:
Agree.